README.md

```diff
@@ -4,18 +4,23 @@
 > Dont forget to [read our contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2018/blob/master/CONTRIBUTING.md)
 
 ## Contents
-- [Repositories Supporting First Timers](#repositories-supporting-first-timers)
-- [Repositories Coding for Beginners](#repositories-coding-for-beginners)
-- [Guides and Resources](#guides-and-resources)
-- [Cli Tools](#cli-tools)
-- [Meetups and Events](#meetups-and-events)
+- [Awesome Hacktoberfest 2018 ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-hacktoberfest-2018-awesomehttpsgithubcomsindresorhusawesome)
```

Comment:
Hey I am not sure about downgrading each content one level. Like, the whole repository is about of hacktoberfest2018, maybe is not necessary to wrap all the content inside this topic.

Reply:
You mean make a new section? What do you want to call it then?

Reply:
No, it doesn't need a new section because the whole repository is about hacktoberfest 2018, so it does not need a section called Hacktoberfest 2018

Reply:
I did not add that. I only added the React projects... let me pull the latest changes you made.

Reply:
Okay! The react projects are already approved, just remove this section and the PR will be approved!